src/site/static/eligibility.liquid

```diff
@@ -2122,10 +2122,46 @@ pageClass: "page-public-assistance"
       underIncomeLimit);
   }
 
+  /*
   function liheapEligible() {
     // TODO
     return false;
   }
+  */
+
+  function liheapEligible() {
+    // https://www.csd.ca.gov/Pages/LIHEAP-Income-Eligibility.aspx
+    //
+    // LIHEAP income limits are set at 60% of State Median Income:
+    //   https://www.acf.hhs.gov/ocs/policy-guidance/liheap-im-2022-04-state-median-income-estimates-optional-use-ffy-2022-and
+    //
+    // Calculating income limit for various household sizes by multiplying the 4-person limit by various values:
+    //   https://www.acf.hhs.gov/sites/default/files/documents/ocs/COMM_LIHEAP_Att1SMITable_FY2023.pdf
+    const grossLimit = new MonthlyIncomeLimit([
+      2700.17,
+      3531.0,
+      4361.83,
+      5192.75,
+      6023.59,
+      6854.43,
+      7010.21,
+      7166.00,
```

Comment:
Either `.00` like this or `.0` as above in `3531.0` is fine. Can you choose one for consistency?